### DISCUSSION TITLE
about softbody fixed

### DISCUSSION THREAD

**Discussion by monkeygame** [NVIDIA-Omniverse/PhysX#41] [Q&A] [answered]
How do I fix partial vertices of softbody?
In flex it is possible to set the mass of each particle, I did not find a similar method in physx

**Comment by ialgniq1974** [comment, marked as answer]
One way to do it is to create an attachment between a static collider and softbody. The overlap region between the collider and softbody will be fixed. You can then hide the collider so that it is not visible in the scene. At this moment, you cannot explicitly specify the vertices to fix in world space. We are working on a tool that allows you to paint the vertices of the softbody and attach those vertices to the world.

**Reply by monkeygame** [reply]
Thank you for your answer, I see the way the attachment is added in the example.
I am looking forward to the tool you mentioned. May everything get better and better
